x/fundraising/abci_test.go

```diff
@@ -30,34 +30,23 @@ func (suite *ModuleTestSuite) TestEndBlockerStandByStatus() {
 }
 
 func (suite *ModuleTestSuite) TestEndBlockerStartedStatus() {
-	suite.keeper.SetAuction(suite.ctx, suite.sampleFixedPriceAuctions[1])
-	err := suite.keeper.ReserveSellingCoin(
-		suite.ctx,
-		suite.sampleFixedPriceAuctions[1].GetId(),
-		suite.sampleFixedPriceAuctions[1].GetAuctioneer(),
-		suite.sampleFixedPriceAuctions[1].GetSellingCoin(),
-	)
-	suite.Require().NoError(err)
+	suite.SetAuction(suite.ctx, suite.sampleFixedPriceAuctions[1])
 
 	auction, found := suite.keeper.GetAuction(suite.ctx, 2)
 	suite.Require().True(found)
 	suite.Require().Equal(types.AuctionStatusStarted, auction.GetStatus())
 
 	totalBidCoin := sdk.NewInt64Coin(suite.sampleFixedPriceAuctions[1].GetPayingCoinDenom(), 0)
 	for _, bid := range suite.sampleFixedPriceBids {
-		bidderAcc, err := sdk.AccAddressFromBech32(bid.Bidder)
-		suite.Require().NoError(err)
-		suite.keeper.SetBid(suite.ctx, bid.AuctionId, bid.Sequence, bidderAcc, bid)
-
-		err = suite.keeper.ReservePayingCoin(suite.ctx, bid.GetAuctionId(), bidderAcc, bid.Coin)
-		suite.Require().NoError(err)
+		suite.PlaceBid(suite.ctx, bid)
 
 		totalBidCoin = totalBidCoin.Add(bid.Coin)
 	}
 
+	receiveAmt := totalBidCoin.Amount.ToDec().Quo(auction.GetStartPrice()).TruncateInt()
```

Comment:
Don't we need to use `QuoTruncate` here, too?

Reply:
Great catch. I missed that.